spec/concepts/todo/create_spec.rb

```diff
@@ -1,7 +1,48 @@
 require 'rails_helper'
 
 RSpec.describe Todo::Create do
-  it "can be spec'd" do
-    raise "Implement me"
+  before(:each) do
+    @todo = Todo.new()
```

Comment:
not sure why each assignment has `()` ? This would be okay without `()`. I'm curious if there's something about it? 😄